### ISSUE TITLE
Package is not compatible with python 3.9

### ISSUE THREAD

**Issue by alipek** [cjaliaga/aioaquarea#39]
On python 3.9 use package code throw error for kw_only argument which can be used from version 3.10

```
    import aioaquarea as aq
  File "/home/user/heat-pump-metrics/venv/lib/python3.9/site-packages/aioaquarea/__init__.py", line 6, in <module>
    from .core import Client
  File "/home/user/heat-pump-metrics/venv/lib/python3.9/site-packages/aioaquarea/core.py", line 21, in <module>
    from .data import (
  File "/home/user/heat-pump-metrics/venv/lib/python3.9/site-packages/aioaquarea/data.py", line 191, in <module>
    @dataclass(kw_only=True)
TypeError: dataclass() got an unexpected keyword argument 'kw_only'
```

**Comment by cjaliaga** [comment]
@alipek thanks for reporting this, I totally missed the support for kw_only was added on 3.10 . Just released 0.5.2 with the original behavior.

**Comment by alipek** [comment]
Good job. Now it works 👍
